src/psyclone/tests/profile_test.py

```diff
@@ -68,6 +68,7 @@ def teardown_function():
 def test_profile_basic(capsys):
     '''Check basic functionality: node names, schedule view.
     '''
```

Comment:
pylint reports that GOKern and GOInvokeSchedule are imported into this file but are not used.

Reply:
deleted imports